### ISSUE TITLE
Sending a request to a different receiver

### ISSUE THREAD

**Issue by teriky** [okfde/froide#367]
This request should handle the following use case regarding the "Klima-Gebäude-Check" campaign.

1. One selects a building to make a request in the overview.
2. The request is sent to the city where the building due to its location belongs to.

If the building isn't a property of the city, the request could not be handled by them.

Now one can either close the request with "Information nicht vorhanden" (information not availabe) or (re)send the request to a different receiver (which is not possible yet).

![froide one](https://user-images.githubusercontent.com/40558729/94684965-107b7180-0329-11eb-8ab7-83f6a442eaab.jpg)

Closing the request seems not reasonable, because the building's state in the overview would not reflect the correct request state.

Sending another request for the same building will be addressed to the same unaccountable city again (which is not desirable).

I suggest a new button "Anderer Empfänger" (other receiver) in conjunction with a selection of a known receiver.

This button/selection combination should be available at the main requesting page too.

![Uploading froide two.jpg…]()

**Comment by arnese** [comment]
Yes! You are absolutely right. We are working on both - making the receiver editable in the request procedure and making it easy to resend the request to a different authority - ideally with only two clicks and with an easy interface. At the moment, the pre-written request asks authorities to forward requests to other authorities if they know where the certificate is instead.
In the meantime while we are fixing this, you can use the "Problem?" button in your request and we will manually resend the request to the authority you point us to.

**Comment by thedudio** [comment]
Hi Arne, 
if somebody is already is working on this issue, maybe he/she should be set as the assignee of this issue (this issue is currenty not assigned to anybody). 
This way its easier to see, which tickets are still in need of somebody who takes care of them.

**Comment by arnese** [comment]
Hi @thedudio, you are absolutely right. As we don't have a lot of ressources for software development at the moment, we don't have proper management of github issues either. But I hope this will change soon!

As for this issue, we have made the receiver of requests editable now, but are still working on making it possible to send faulty requests to different authorities after they were sent.
